reviewnb_demo/demo.ipynb

```diff
@@ -85,71 +85,35 @@
        "      <td>6.0</td>\n",
```

Comment:
<p>looks like you can only leave comments on a single block of notebook code, rather than being able to leave  comments on specific lines of code.</p><p>As long as we keep our cells short and single-purpose - this shouldn't be a problem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/vyshaal/reviewnb-demo/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>